fnConsensus/store.go

```diff
@@ -5,6 +5,10 @@ import dbm "github.com/tendermint/tendermint/libs/db"
 const reactorStateKey = "fnConsensusReactor:state"
 
 func loadReactorState(db dbm.DB) (*ReactorState, error) {
+	if db == nil {
+		return NewReactorState(), nil
+	}
```

Comment:
Nah, fix this in `FnConsensusReactor.OnStart()`, on a non-validator don't call `loadReactorState`